tools/c7n_azure/c7n_azure/function_package.py

```diff
@@ -96,14 +107,10 @@ def get_function_config(self, policy):
             binding['schedule'] = policy['mode']['schedule']
 
         elif mode_type == CONST_AZURE_EVENT_TRIGGER_MODE:
-            binding['type'] = 'httpTrigger'
-            binding['authLevel'] = 'function'
+            binding['type'] = 'queueTrigger'
+            binding['connection'] = 'AzureWebJobsStorage'
```

Comment:
Do we support the scenario where the user can specify the queue's SA? (default could be to use the same SA as the Function App for logs and internal stuff)

Reply:
No, we just use the provided storage account.